specification/storagepool/resource-manager/Microsoft.StoragePool/preview/2020-03-15-preview/storagepool.json

```diff
@@ -195,7 +195,7 @@
               "$ref": "#/definitions/DiskPool"
             }
           },
-          "202": {
+          "201": {
```

Comment:
In my test, your server is still returning 202, but here you change to 201, which will cause misunderstanding for SDK.

Reply:
@Juliehzl we were waiting for the private builds of the PSH module and az cli to be updated with the new status code before we make the code change.

The server should be returning the right status code now.